### DISCUSSION TITLE
Reading my files on Karavan web : docker

### DISCUSSION THREAD

**Discussion by MN-Dheeraj** [apache/camel-karavan#1246] [Q&A]
### Ask a question

I need my processor code on karavan to read the content of the file. Is there a way to place it some where and read it into my java code

### Variant

Web Application

### Container Management (if applicable)

Docker

### Operating System (if applicable)

Windows

### Version

4.4.0

**Comment by davsclaus** [comment]
you can create a file in the project, and add that file to the classpath as a dependency, and you can read it from classpath in java code

**Reply by MN-Dheeraj** [reply]
Can you please elaborate what mean by adding file to classpath as dependency? Can't I just read the file in the project by giving it's location?
Is there a process to add the location to application.properties or something?
A document that explains this(w.r.t Karavan-web) would be helpful.

**Reply by MN-Dheeraj** [reply]
Also, why do you close the issue with out confirmation from the ticket author?
